backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientDataResolver.java

```diff
@@ -25,4 +35,30 @@ public ApiFacility getFacility(Person p) {
     Facility f = p.getFacility();
     return f == null ? null : new ApiFacility(f);
   }
+
+  public CompletableFuture<TestResultDeliveryPreference> getTestResultDelivery(
+      Person person, DataFetchingEnvironment dfe) {
+    DataLoaderRegistry registry = ((GraphQLContext) dfe.getContext()).getDataLoaderRegistry();
+    DataLoader<UUID, PatientPreferences> patientPreferencesLoader =
+        registry.getDataLoader(PatientPreferences.DATA_LOADER);
+    if (patientPreferencesLoader != null) {
+      return patientPreferencesLoader
+          .load(person.getInternalId())
+          .thenApply(PatientPreferences::getTestResultDelivery);
+    }
+    throw new IllegalStateException("No patient preferences data loader found");
+  }
+
+  public CompletableFuture<String> getPreferredLanguage(
```

Comment:
Not to belatedly nitpick, but this seems a tad duplicative. 😁

Reply:
Addressed in #1338 ;)